lib/gasman.gi

```diff
@@ -27,6 +27,16 @@ InstallGlobalFunction( CollectGarbage, function( full )
     fi;
 end );
 
+#############################################################################
+##
+#F  GasmanDisableMemoryLimits( )
+##
+##  Disable limits on GASMAN workspace size.
+##
+InstallGlobalFunction("GasmanDisableMemoryLimits", function()
```

Comment:
I think Boehm GC also honors `SyStorKill`, and in the future, perhaps also the Julia GC might. So how about calling this just `DisableMemoryLimits`, and talk about "workspace size", not "GASMAN workspace size"?

Reply:
I wasn't positive what other GCs might do, I'm happy to make this a global option